pkg/lockservice/lock_table_remote.go

```diff
@@ -264,3 +267,7 @@ func (l *remoteLockTable) maybeHandleBindChanged(resp *pb.Response) error {
 	l.bindChangedHandler(*newBind)
 	return ErrLockTableBindChanged
 }
+
+func isRetryError(err error) bool {
```

Comment:
Come on ...    You have to catch the error and test on error code.

Reply:
Unable to catch the error code, this may be the remote serialization over the error, this time there is no original type, only the error string.

Reply:
No way -- remote serialization serialize error code as well.    If it is not a moerr (3rd party), you must convert it to moerr immediately.

Don't tell me a not found on a table called "timeout" will trigger this code ...

Reply:
Should you just test RPCTimeout for this case?

Reply:
> No way -- remote serialization serialize error code as well. If it is not a moerr (3rd party), you must convert it to moerr immediately.
> 
> Don't tell me a not found on a table called "timeout" will trigger this code ...

There is no table name used inside the lockservice.  It's almost always a net error.

Reply:
OK, let me try convert net error to moerr